client/my-sites/checkout/composite-checkout/wpcom/components/wp-order-review-line-items.js

```diff
@@ -310,23 +310,14 @@ WPOrderReviewLineItems.propTypes = {
 };
 
 const WPOrderReviewList = styled.ul`
-	margin: -10px 0 10px 0;
-	padding: 0;
+	margin: 10px 0;
```

Comment:
What is the purpose of these CSS changes? It's not mentioned in the commit, and it seems pretty significant since these styles are used for the line items in the review step.

Reply:
Sorry, the `-10px` top margin was used to visually make the list items "nest" under the step title. In the new design, it's been [replaced by the `domainUrl`](https://github.com/Automattic/wp-calypso/pull/41063/files#diff-8503f114a52569d2dee502c3a31acb6bR74).